easybuild/easyblocks/p/pdt.py

```diff
@@ -53,14 +53,16 @@ def configure_step(self):
 
         # determine values for compiler flags to use
         known_compilers = {
+            toolchain.DUMMY: '-GNU',  # Assume that dummy toolchain uses a system-provided GCC
             toolchain.GCC: '-GNU',
             toolchain.INTELCOMP: '-icpc',
         }
         comp_fam = self.toolchain.comp_family()
         if comp_fam in known_compilers:
-            self.compiler_opt = known_compilers[comp_fam]
+            compiler_opt = known_compilers[comp_fam]
         else:
             raise EasyBuildError("Compiler family not supported yet: %s" % comp_fam)
+        self.cfg.update('configopts', compiler_opt)
```

Comment:
Apparently, no.  So, one could argue that this flag doesn't matter as the easyblock has worked fine until now, but I can't say for sure which consequences omitting this flag really has. I guess that `configure` always picked up GCC, regardless of the actual toolchain...